src/workflow.scss

```diff
@@ -0,0 +1,1147 @@
+/* GOOGLE POPUP */
+.workflow-extension-google-popup {
+  min-width: 450px !important;
```

Comment:
Hi @DimitrijeMihic woah, why so many importants??

Reply:
Hi @sagefright , so far we noticed the style of the page itself can affect the style of the elements we add from the extension to the DOM. For example if the page has set ` margin-bottom: 20px !important;` for every paragraph, it will override the style of paragraphs in our elements. Adding !important was a way to make sure the style will remain the same on all websites.

Reply:
OK, thanks. This is something we can look at in the future!